### PR TITLE
Rename ViewHelpers to TemplateHelpers

### LINKED ISSUE #125: Rename View Helpers to Template Helpers
@frm I think we should rename View Helpers to Template Helpers. We are calling those files _templates_, not _views_. I think the only reason we called it views is because we were thing about Phonix. What do you say?

### PR DESCRIPTION
**Pull request by hgg**
Closes #125

### CODE REVIEW

File: lib/still/preprocessor/renderer.ex [hunk 1]
```diff
@@ -58,7 +58,7 @@ defmodule Still.Preprocessor.Renderer do
         Module.concat(["R#{Enum.random(0..100_000)}" | [@preprocessor | name]])
       end
 
-      defp create_view_renderer(name, content, metadata) do
+      defp create_template_renderer(name, content, metadata) do
```
Comment (hgg) [inline lib/still/preprocessor/renderer.ex:61]: Not sure about this change. LMK if this function name wasn't supposed to change.

**Comment by hgg** [comment]
I'm believe that the current approach is a breaking change due to the config field change. Let me know if you want me to come up with a backwards compatible way.

File: lib/still/preprocessor/renderer.ex (hunk 1 above)
Comment (gabrielpoca) [reply lib/still/preprocessor/renderer.ex:61]: I think this change makes sense 👍

**Review by gabrielpoca — APPROVED** [review]
I think this is fine. It's too early in my opinion to start introducing backward compatibility in this feature. We are also planning another change that's going to break the API in a way that's not backward compatible xD

**Pull request merged**